doc/api/errors.md

```diff
@@ -2063,12 +2063,6 @@ attempt to set the `secureProtocol` explicitly. Use one mechanism or the other.
 
 An attempt was made to renegotiate TLS on a socket instance with TLS disabled.
 
-<a id="ERR_TLS_REQUIRED_SERVER_NAME"></a>
-### `ERR_TLS_REQUIRED_SERVER_NAME`
-
-While using TLS, the `server.addContext()` method was called without providing
-a host name in the first parameter.
```

Comment:
Can you move this to the “Legacy Node.js error codes” section below?

Reply:
Done-ish. I wasn't sure how to fill the YAML added-removed part, so I did not add it. I welcome anyone to fill it or to provide me guidance on how to do it properly,

Reply:
Yeah, it’s unfortunate that we (or, to be completely honest, I) did not do this for errors when we started introducing YAML metadata.

If you want to add it, this is the “right” one:

```
<!-- YAML
added: v9.0.0
removed: REPLACEME
-->
```

(If not, I don’t think there’s any harm done by keeping it left out either.)

Reply:
Thanks! Added in following commit.